app-server/app/src/main/scala/thumbnail/QueryParameters.scala

```diff
@@ -0,0 +1,32 @@
+package com.azavea.rf.thumbnail
+
+import java.util.UUID
+import java.sql.Timestamp
+import java.time.Instant
+
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.{Directive1, Rejection}
+import akka.http.scaladsl.server.directives.ParameterDirectives.parameters
+
+import com.azavea.rf.utils.queryparams._
+
+/** Case class representing all /thumbnail query paremeters */
+case class ThumbnailQueryParameters(
+  sceneId: Option[UUID]
+)
+
+/** Combined all relevant query parameters for /thumbnails */
+case class CombinedThumbnailQueryParameters(
```

Comment:
There's no reason to separate out the combined and specific query parameters when there is only one set as it is here. The other endpoints have combined/specific because they have additional filters that are generic.